.travis.yml

```diff
@@ -1,19 +1,41 @@
 language: python
-
 cache: pip
+env:
+  - PR_AUTHOR=${TRAVIS_PULL_REQUEST_SLUG::-15}
 
-sudo: required
-
-before_install:
- - gem install asciidoctor
- - gem install asciidoctor-diagram
-
-install:
- - pip3 install pyyaml
- - pip3 install aura.tar.gz
-
-script:
- - python3 build.py --distro openshift-enterprise --product "OpenShift Container Platform" --version 4.10 --no-upstream-fetch && python3 makeBuild.py
+jobs:
+  include:
+    - stage: build
+      before_install:
+        - gem install asciidoctor
+        - gem install asciidoctor-diagram
+      install:
+        - pip3 install pyyaml
+        - pip3 install aura.tar.gz
+      script:
+        - python3 build.py --distro openshift-enterprise --product "OpenShift Container Platform" --version 4.10 --no-upstream-fetch && python3 makeBuild.py
+    - # stage name not required, will continue to use `build`
+      before_install:
+        - gem install asciidoctor
+        - gem install asciidoctor-diagram
+      install:
+        - pip3 install pyyaml
+        - pip3 install aura.tar.gz
+      script:
+        - python3 build.py --distro openshift-dedicated --product "OpenShift Dedicated" --version 4 --no-upstream-fetch && python3 makeBuild.py
+    - # stage name not required, will continue to use `build`
+      before_install:
+        - gem install asciidoctor
+        - gem install asciidoctor-diagram
+      install:
+        - pip3 install pyyaml
+        - pip3 install aura.tar.gz
+      script:
+        - python3 build.py --distro openshift-rosa --product "Red Hat OpenShift Service on AWS" --version 4 --no-upstream-fetch && python3 makeBuild.py
+    - stage: automerge
```

Comment:
@gaurav-nelson does the automerge stage only trigger if all of the previous stages report a success?

Reply:
@pneedle-rh Yes. From [Travis Docs](https://docs.travis-ci.com/user/build-stages/):
> In the simplest and most common use case, you can now make one job run only if several other, parallel jobs have completed successfully.

Reply:
Excellent! Thank you for confirming that.